src/handlers/gestureHandlerCommon.ts

```diff
@@ -83,7 +83,7 @@ export type GestureTouchEvent = {
   handlerTag: number;
   numberOfTouches: number;
   state: ValueOf<typeof State>;
-  eventType: EventType;
+  eventType: TouchEventType;
```

Comment:
I guess we cannot rename this field to `touchEventType` because it would be a brekaing change in API, is that true?

Reply:
Correct.